### PR TITLE
Changing + fixing the custom logging function behaviour

### PR DESCRIPTION
**Pull request by marismmm**
(description of the change)

#### Pull Request Checklist ##
- [ ] Changes are complete (if posting work-in-progress code, prefix your pull request title with '[WIP]'
until you can check this box.
- [ ] Unit tests pass (`pytest`)
- [ ] Were system tests modified? If so did you run these at least 5 times to account for the variation across runs?
- [ ] System tests pass (`pytest test/system`)
- [ ] Passed code style checking (`./style-check.sh`)
- [ ] You have considered writing a test
- [ ] Updated major/minor version in `sockeye/__init__.py`. Major version bump if this is a backwards incompatible change.
- [ ] Updated CHANGELOG.md

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.109]
+### Changed
```
Comment (tdomhan) [inline CHANGELOG.md]: Maybe prefix this as `Fixed` instead of `Changed`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ Note that Sockeye has checks in place to not translate with an old model that wa
...
+- Restricted the custom logging function to accept only a dictionary and a compulsory global_step parameter. Also fixed its usage by explicitly prefixing training and validation metrics.
```
Comment (tdomhan) [inline CHANGELOG.md]: Maybe maybe the prefixes explicit: "Fixed the metric names by prefixing training metrics with 'train-' and validation metrics with 'val-'".

Comment (marismmm) [reply CHANGELOG.md]: Addressed.

File: CHANGELOG.md (hunk 1 above)
Comment (marismmm) [reply CHANGELOG.md]: Addressed.

**Pull request merged**